File: app/src/main/java/com/drivingschoolrwandaapp/models/request/ResetPasswordRequest.java
package com.drivingschoolrwandaapp.models.request;

import com.google.gson.annotations.SerializedName;

public class ResetPasswordRequest {
    @SerializedName("token")
    private String token;

    @SerializedName("newPassword")
    private String newPassword;

    @SerializedName("confirmPassword")
    private String confirmPassword;

    public ResetPasswordRequest(String token, String newPassword, String confirmPassword) {
        this.token = token;
        this.newPassword = newPassword;
        this.confirmPassword = confirmPassword;
    }

    public String getToken() { return token; }
    public String getNewPassword() { return newPassword; }
    public String getConfirmPassword() { return confirmPassword; }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/request/ResetPasswordRequest.kt
package com.drivingschoolrwandaapp.models.request

import com.google.gson.annotations.SerializedName

data class ResetPasswordRequest(
    @SerializedName("token") var token: String = "",
    @SerializedName("newPassword") var newPassword: String = "",
    @SerializedName("confirmPassword") var confirmPassword: String = ""
)
